biocrnpyler/core/chemical_reaction_network.py

```diff
@@ -65,7 +65,7 @@ def reactions(self, reactions):
             raise AttributeError("The reactions in a CRN cannot be removed or modified. New reactions can be added with CRN.add_reactions(...).")
 
     
-    def add_species(self, species, copy_species = True):
+    def add_species(self, species, copy_species = True, compartment = None):
```

Comment:
PEP8: as long as you are making a change here (and elsewhere?), you might as well use PEP8 rules (no spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter).

Reply:
Yes, will use autopep8 formatter to all files.